bin/node-template/runtime/src/lib.rs

```diff
@@ -812,7 +812,17 @@ impl pallet_sudo::Trait for Runtime {
 }
 
 parameter_types! {
-	pub const EthBackingModuleId: ModuleId = ModuleId(*b"da/backi");
+	pub const CrabIssuingModuleId: ModuleId = ModuleId(*b"da/crabi");
+}
+impl darwinia_crab_issuing::Trait for Runtime {
+	type Event = Event;
+	type ModuleId = CrabIssuingModuleId;
+	type RingCurrency = Ring;
+	type WeightInfo = ();
+}
+
+parameter_types! {
+	pub const EthBackingModuleId: ModuleId = ModuleId(*b"da/ethbk");
```

Comment:
Changing the EthBackingModuleId could cause related account address being changes.

Related account address might have tokens in genesis.

We need to print these module account id and do some check on Crab.

Reply:
Yes.